contributing/contribute-code.zh.md

```diff
@@ -12,6 +12,13 @@ Apache Flink 是一个通过志愿者贡献的代码来维护、改进和扩展
 
 {% toc %}
 
+## 寻找可贡献的方内容
+
+如果你已经有好的想法可以贡献，可以直接参考下面的 "代码贡献步骤"。
```

Comment:
To not get blocked by issue FLINK-18496, a quote pair is used instead of an anchor link, which is a bit different than the english version.